Code/GraphMol/Wrap/StereoGroup.cpp

```diff
@@ -40,6 +40,15 @@ StereoGroup *createStereoGroup(StereoGroupType typ, ROMol &mol,
     cppAtoms.push_back(mol.getAtomWithIdx(v));
     ++beg;
   }
+  python::stl_input_iterator<unsigned int> bbeg(bondIds), bend;
+  while (bbeg != bend) {
+    unsigned int v = *bbeg;
+    if (v >= mol.getNumBonds()) {
+      throw_value_error("bond index exceeds mol.GetNumBonds()");
+    }
+    cppBonds.push_back(mol.getBondWithIdx(v));
+    ++bbeg;
+  }
   auto *sg = new StereoGroup(typ, cppAtoms, cppBonds, readId);
```

Comment:
Before adding the stereo group, you should check to see if there is at least one bond or atom provided and thrown a `ValueError` if not. As written, this allows an empty StereoGroup to be created, which doesn't make much sense.